_test_unstructured_client/test__decorators.py

```diff
@@ -81,6 +80,13 @@ def test_integration_split_pdf_has_same_output_as_non_split(
     assert resp_split.status_code == resp_single.status_code
 
     # Difference in the parent_id is expected, because parent_ids are assigned when element crosses page boundary
-    diff = DeepDiff(t1=resp_split.elements, t2=resp_single.elements,
-                    exclude_regex_paths=r"root\[\d+\]\['metadata'\]\['parent_id'\]")
+    diff = DeepDiff(
+        t1=resp_split.elements,
+        t2=resp_single.elements,
+        exclude_regex_paths=[
+            r"root\[\d+\]\['metadata'\]\['parent_id'\]",
+            # TODO: (Marek Połom) - Remove page number pattern after page numbering parameter is added
+            r"root\[\d+\]\['metadata'\]\['page_number'\]",
```

Comment:
Do you know when will the page numbering parameter be added?

Reply:
Ok - then it would be good idea to add here CORE number or link, so it's easy to find what the comment talks about